docs/source/examples/petstore/petstore.all.rego.compiled

```diff
@@ -68,18 +74,23 @@ allow {
     seal_list_contains(input.subject.groups, `customers`)
     input.verb == `buy`
     re_match(`petstore.pet`, input.type)
-input.status == "available"
+    input.status == "available"
 }
 allow {
     seal_list_contains(input.subject.groups, `breeders_maltese`)
     input.verb == `buy`
     re_match(`petstore.pet`, input.type)
-input.status == "reserved"
-input.breed == "maltese"
+    input.status == "reserved"
+    input.breed == "maltese"
```

Comment:
These ctx input values input.status and input.breed, et al, should be
input.ctx.status and input.ctx.breed, etc.  That's how the middleware will populate the OPA input doc.

Reply:
i'll create a separate issue to deal with that - please do not fix that in this PR

Reply:
https://github.com/infobloxopen/seal/issues/81

Reply:
Forgot to mention, input.ctx will be an array, so it should be input.ctx.status[_] == "reserved", etc.

Reply:
why is input.ctx an array?  let's not discuss it here -  please comment in #81